docs/modules/localstack.md

```diff
@@ -62,6 +62,26 @@ Testcontainers will inform Localstack of the best hostname automatically, using
 
 * Other usage scenarios, such as where the Localstack container is used from both the test host and containers on a custom network are not automatically supported. If you have this use case, you should set `HOSTNAME_EXTERNAL` manually.
 
+## AWS Lambda and Networking
+
+When a test using Localstack triggers a [AWS Lambda function](https://aws.amazon.com/lambda/), Localstack [starts a new docker container](https://docs.localstack.cloud/localstack/lambda-executors/#docker) to run the Lambda code. This might cause problems if your Lambda function calls other AWS services in your Localstack rig. To fix this we have to make sure these Lambda containers are in the same network as the rest of our containers.
+
+```java
+Network shared = Network.newNetwork()
+String networkName = ((NetworkImpl)shared).getName();
+
+DockerImageName localstackImage = DockerImageName.parse("localstack/localstack:0.11.3");
+
+@Rule
+public LocalStackContainer localstack = new LocalStackContainer(localstackImage)
+        .withServices(LAMBDA)
+        .withEnv("LAMBDA_DOCKER_NETWORK", networkName)
+        .withNetwork(network)
+        .withNetworkAliases("localstack")
```

Comment:
can we move this to a test [here](https://github.com/testcontainers/testcontainers-java/blob/master/modules/localstack/src/test/java/org/testcontainers/containers/localstack/LocalstackContainerTest.java), please? 

should be imported using
```
<!--codeinclude-->
[AWS SDK V1](../../modules/localstack/src/test/java/org/testcontainers/containers/localstack/LocalstackContainerTest.java) inside_block:with_aws_sdk_v1
<!--/codeinclude-->
```